readthedocs/oauth/managers.py

```diff
@@ -10,12 +14,15 @@
 DEFAULT_PRIVACY_LEVEL = getattr(settings, 'DEFAULT_PRIVACY_LEVEL', 'public')
 
 
-class GithubProjectManager(models.Manager):
-    def create_from_api(self, api_json, user, organization=None,
-                        privacy=DEFAULT_PRIVACY_LEVEL):
+class RemoteRepositoryManager(models.Manager):
```

Comment:
These should inherit from the privacy related managers to get proper filtering. Probably the RelatedProject ones.